sysbuild/Kconfig.sysbuild

```diff
@@ -25,13 +25,6 @@ config PM_MCUBOOT_PAD
 	  different padding size, then this value must be updated to correspond
 	  to the value in the static partition file.
 
-config PM_OVERRIDE_EXTERNAL_DRIVER_CHECK
```

Comment:
Unrelated change perhaps? But if it is intentional consider also removing the commented out code in lines 30-42